src/hnc_app.erl

```diff
@@ -20,7 +20,19 @@
 -export([stop/1]).
 
 start(_, _) ->
+	%% Suppress worker_sup shutdown reports, they are expected if a worker crashes.
+	_=logger:add_primary_filter(
+		?MODULE,
+		{
+			fun
+				(#{meta:=#{hnc:=#{module:=hnc_worker_sup}}, msg:={report, #{label:={supervisor, shutdown}}}}, undefined) -> stop;
+				(_, undefined) -> ignore
+			end,
+			undefined
+		}
+	),
```

Comment:
What is this for?

Reply:
See below.

Reply:
Ah, it's even there in a comment, should read more carefully ;)